plugins/generator-1.17.1/forge-1.17.1/procedures/entity_lookpos_z.java.ftl

```diff
@@ -0,0 +1,3 @@
+(${input$entity}.level.clip(new ClipContext(${input$entity}.getEyePosition(1f),${input$entity}.getEyePosition(1f)
+    .add(${input$entity}.getViewVector(1f).scale(${input$maxdistance})), ClipContext.Block.${field$block_mode},
+    ClipContext.Fluid.${field$fluid_mode}, ${input$entity})).getBlockPos().getZ())
```

Comment:
You should specify default values here as in the old generators as older workspaces may not have these fields set yet